clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesHandler.java

```diff
@@ -45,10 +45,20 @@
 
   private final ArtifactReplacer artifactReplacer;
 
+  protected String dockerImageBinding;
+
   protected KubernetesHandler() {
     this.artifactReplacer = new ArtifactReplacer(artifactReplacers());
   }
 
+  protected void setDockerImageBinding(String dockerImageBinding) {
```

Comment:
This method was defined so the concrete class can assign the value from config properties.